lib/request/request.rb

```diff
@@ -0,0 +1,152 @@
+        #
+        # - user [Organization/Project object, default None]: Organization or Project object. Not necessary if StarkBank.user
+        #     was set before function call
+        # - query [json, default None]: Query parameters. ex: {"limit": 1, "status": paid}
+        #
+        # Return:
+        #
+        # - list of resources jsons with updated attributes
+
+        def self.post(path:, payload:, query: nil, user: nil)
+            content = StarkBank::Utils::Rest.post_raw(
+                path: path,
+                query: query,
+                payload: payload,
+                user: user,
+                prefix: "Joker",
+                raiseException: false
+            )
+        end
+
+        # # Update any StarkBank resource
+        #
+        # Send a json with parameters of a single StarkBank resource object and update it
+        #
+        # Parameters (required):
+        #
+        # - path [string]: StarkBank resource's route. ex: "/invoice/5699165527090460"
+        # - body [json]: request parameters. ex: {"amount": 100}
+        #
+        # Parameters (optional):
+        #
+        # - user [Organization/Project object, default None]: Organization or Project object. Not necessary if StarkBank.user
+        #     was set before function call
+        #
+        # Return:
+        # - json of the resource with updated attributes
+
+        def self.patch(path:, payload:, query: nil, user: nil)
+            content = StarkBank::Utils::Rest.patch_raw(
+                path: path,
+                query: query,
+                payload: payload,
+                user: user,
+                prefix: "Joker",
+                raiseException: false
+            )
+        end
+
+        # # Put any StarkBank resource
+        #
+        # Send a json with parameters of a single StarkBank resource object and create it, if the resource alredy exists,
+        # you will update it.
+        #
+        # Parameters (required):
+        #
+        # - path [string]: StarkBank resource's route. ex: "/split-profile"
+        # - body [json]: request parameters. ex: {"profiles": [{"interval": day, "delay": "1"}]}
+        #
+        # Parameters (optional):
+        #
+        # - user [Organization/Project object, default None]: Organization or Project object. Not necessary if StarkBank.user
+        #     was set before function call
+        #
+        # Return:
+        # - json of the resource with updated attributes
+
+        def self.put(path:, payload:, query: nil, user: nil)
+            content = StarkBank::Utils::Rest.put_raw(
+                path: path,
+                query: query,
+                payload: payload,
+                user: user,
+                prefix: "Joker",
+                raiseException: false
+            )
+        end
+
+        # # Delete any StarkBank resource
+        #
+        # Send a json with parameters of a single StarkBank resource object and delete it
```

Comment:
Aqui normalmente é pela rota só (path e/ou queryString)
```suggestion
        # Send parameters of StarkBank resources and delete them
```